Langs/0413.lang

```diff
@@ -57,3 +63,18 @@ Invalid Release Tags Received!=Ongeldige release-tags ontvangen!
 Invalid Release Types Received!=Ongeldige release-types ontvangen!
 Update Available=Update beschikbaar
 An Update is Available, would you like to download it?=Er is een update beschikbaar. Wil je deze downloaden?
+Done=Klaar
+Loading WMIC=WMIC laden...
+Warning=Waarschuwing
+[Descriptions]
+Architecture=De gegevenslengte waar je processor en besturingssysteem mee werken. Als je hier een 32-bits besturingssysteem ziet staan, moet je de schijf wissen en Windows 11 opnieuw installeren. Bij een 32-bits processor zul je deze moeten vervangen (indien mogelijk).
+Boot=De methode die het moederbord gebruikt om Windows op te starten. Een "Legacy" status kun je oplossen door de instellingen van je BIOS/UEFI te veranderen. Zie de handleiding van je moederbord.
+CPU Name=Het soort processor dat je in je pc hebt geinstalleerd. Compatibiliteit kan variëren en wordt (waarschijnlijk) door Microsoft bepaald. Indien niet compatibel, kun je overwegen de processor te vervangen. Dit kan alleen bij een desktop-pc. Bij een laptop is dit niet mogelijk.
```

Comment:
Shouldn't there be this i with 2 dots in "geinstalleerd"? Otherwise looks fine.

Reply:
Good finding. I’ve corrected it.